### PR TITLE
Support Unknown Parameters in INIT Chunk

### PR DESCRIPTION
**Pull request by Sean-Der**
Start to implement section-3.2.1 and respect chunks with a action of Skip. We don't properly report the values back yet.

### CODE REVIEW

**Comment by Sean-Der** [comment]
If someone has the availability mind looking at this? Soon Chrome is going to roll out with this new param (that we don't support yet)

Pion is going to fail to negotiate SCTP then. I want to get this out so we don't see DataChannels failing.

**Comment by Sean-Der** [comment]
Test failure is ok/expected. We are breaking public API by not having unmarshal errors anymore

File: chunk_init_common.go [hunk 1]
```diff
@@ -91,18 +91,21 @@ func (i *chunkInitCommon) unmarshal(raw []byte) error {
 	remaining := len(raw) - offset
 	for remaining > 0 {
 		if remaining > initOptionalVarHeaderLength {
-			pType, err := parseParamType(raw[offset:])
-			if err != nil {
+			pHeader := paramHeader{}
```
Comment (edaniels) [inline chunk_init_common.go:94]:
```suggestion
			var pHeader paramHeader
```

**Review by edaniels — APPROVED** [review]
LGTM. Might be helpful to comment on report actions that we don't actually do that and/or point to a gh issue

**Pull request merged**